libraries/dspxUtils/bidderUtils.js

```diff
@@ -21,7 +21,7 @@ export function fillUsersIds(bidRequest, payload) {
       did_pubcid: 'pubcid.org',
       did_cruid: 'criteo.com',
       did_tdid: 'adserver.org',
-      did_pbmid: 'regexp:[esp\.]*pubmatic\.com',
+      did_pbmid: 'regexp:[esp\\.]*pubmatic\\.com',
```

Comment:
this is not equivalent, do we know if it's correct? cc @onlsol

Reply:
@dgirardi pretty clear it is wrong on further inspection

'regexp:^(?:esp\\.)?pubmatic\\.com$'
should catch pubmatic.com and esp.pubmatic.com, they are looking for either

Reply:
the previous version does look wrong, but I don't share your confidence in asserting that what their backend does is perfectly clear :)